lib/utils/config/index.js

```diff
@@ -23,6 +23,7 @@ function createConfig() {
     userId: null, // currentUserId
     frameworkId: initialSetup.generateFrameworkId(),
     trackingDisabled: initialSetup.configureTrack(), // default false
+    enterpriseDisabled: false,
```

Comment:
Is this a config we keep in the codebase or do we read that from a config file (`.serverlessrc` or smth. like that) later on?

Reply:
oh. good catch. Accidentally removed in c17196aef76fd6392361bac86c017e6245827b46 I've readded it

Reply:
Sounds good 👍

Just another question real quick: Do we keep it globally enabled for now and change it to be opt-out via `.serverlessrc` later on? Just asking since this is a hard-coded config where (almost) everything config-related is loaded from disk.